### DISCUSSION TITLE
question

### DISCUSSION THREAD

**Discussion by c2xusnpq6** [AdguardTeam/AdGuardHome#3388] [Q&A]
Can I run [Outline](https://github.com/Jigsaw-Code/outline-server) and [Adguard Home](https://github.com/AdguardTeam/AdGuardHome) in the same VM at the same time? (I am not a professional in this field...😗)

(I'm using Google Cloud Platform)

**Comment by c2xusnpq6** [comment]
https://github.com/Jigsaw-Code/outline-server/issues/941

**Comment by c2xusnpq6** [comment]
Is there any way to make these two software coexist? Because Google Cloud reminds me that they conflict...

**Comment by ameshkov** [comment]
Tbh, I don't know, this depends on whether outline allows to customize the DNS settings.
